omod/src/main/java/org/openmrs/module/kenyaemr/fragment/controller/report/AdxViewFragmentController.java

```diff
@@ -77,19 +72,21 @@ public class AdxViewFragmentController {
     protected final Log log = LogFactory.getLog(getClass());
 
     private LocationService locationService;
-    public String SERVER_ADDRESS = "http://41.204.187.152:9721/api/";
+    public String SERVER_ADDRESS = "https://openhimapi.kenyahmis.org/rest/api/IL/MOH_731/test";
```

Comment:
Could we have this as a global property?

Reply:
This is already a global property. We use this as a default in case the property value is not set.